### ISSUE TITLE
Snip tool (inlining tool) usability

### ISSUE THREAD

**Issue by codicusmaximus** [notsecure/uTox#877]
On Windows (not sure about other platforms) when you click the green button to the right of the text entry box to make the screen go gray and allow the user to click drag a region of the screen to screenshot and inline there are two important things that need to happen to make that feature usable. 

1) Make it cancel-able with the Esc key.  Right now you must go through with the operation and transfer a blank png file
2) Hide the chat client.  This is very important because if you click that button even when the chat client is behind other things it pops up to the front over what you are trying to take a screenshot of to begin with.  So you must remember to rearrange everything on your screen such that your chat client won't get in the way.

**Comment by tsudoko** [comment]
I think there was a separate issue for 2, can't find it right now. Anyway, hiding the client should be optional to be able to take screenshots of the client.

**Comment by codicusmaximus** [comment]
That's a good point, but I don't think taking and sending snips of the client itself is a very common use case.  Maybe holding Alt Shift or Ctrl or something like that when clicking the green box can keep the client up.

**Comment by fcore117** [comment]
now because of UE4 usage every day i would need too that uTox should hide when pressing snip button.

**Comment by kriswema** [comment]
1. #596: see [this](https://github.com/notsecure/uTox/issues/596#issuecomment-58765536) - doesn't it work, or why wasn't it implemented?
2. #715

**Comment by codicusmaximus** [comment]
Not sure if this issue should be closed due to those other two, or if they should just be merged into this one.  Is there any kind of issue upvoting on GitHub?  I feel like since there are a lot of open issues, certain ones that are pretty important but relatively easy to implement can easily get buried in the noise.

**Comment by atgote** [comment]
Just want to thank codicusmaximus for this issue. I was going to add this (actually, the second one) to the list when noticed this. Awaiting this to be fixed...

**Comment by Doom032** [comment]
I agree with the escape key to cancel but I'm not sure about defaulting to hiding the client. I screenshot things in the client pretty often, and I think ctrl+clicking for that would not be obvious enough. Nevertheless I've also been annoyed by µTox being in the way when wanting to screenshot something else, so I understand there should be a way to do this apart from alt+tabbing around.

Maybe right mouse button on the screenshot button to open a context menu with "screenshot (don't close client)" as an option? Might not be obvious enough either, though. An extra button is possible but probably overkill. I'm not sure about how to do this.

**Comment by fcore117** [comment]
perhaps auto minimize to taskbar(should work all os-es) when clicking snip button?

**Comment by GrayHatter** [comment]
Tool tip: "shift + click" will hide utox

**Comment by tsudoko** [comment]
I think a tooltip needs to describe what the button is for as well, it's not really obvious.

**Comment by lunixgeek** [comment]
I agree that some of the buttons are obvious, especially the ones at the bottom of the text entry screen.
Also, perhaps <Enter> shouldn't sent the message but instead create a blank line for new paragraph.

**Comment by GrayHatter** [comment]
blocked by https://github.com/GrayHatter/uTox/pull/14 please reopen this issue here https://github.com/GrayHatter/uTox/
